pkg/invariants/testframework/legacytestframeworkinvariants/alerts.go

```diff
@@ -60,6 +62,20 @@ func testAlerts(events monitorapi.Intervals,
 		}
 	}
 
+	if kubeClient == nil {
+		kubeClient, err = framework.LoadClientset(true)
```

Comment:
plumb what's needed through.  there's a lot more flexibility after the refactor and this global method doesn't age well.

Reply:
Done